keps/sig-scheduling/3902-decoupled-taint-manager/README.md

```diff
@@ -168,19 +171,28 @@ Since there are no new API changes, we can skip Alpha and go to Beta directly.
 
 * Support `kube-controller-manager` flag `--controllers=-taint-eviction-controller` to disable the default `TaintEvictionController`.
 * Unit and e2e tests completed and passed.
-* Performance and scalability tests to verify there is non-negligible performance degradation in node taint-based eviction.
```

Comment:
Why is it fine to exclude this requirement?

Maybe at least add a few words explaining that this change is not impacting performance and why. I do not remember a history why it was added here in the first place, maybe there were some concerns regarding implementation? cc: @alculquicondor

Reply:
I don't think there were any specific concerns, but it doesn't seem appropriate to remove without explaining why.

Reply:
I'm sorry about that. Some discussions are in [this thread](https://github.com/kubernetes/enhancements/pull/4392#discussion_r1766450515). Should we add some words in KEP to explain why it was removed? 

How about this?

```
In the past, the taint-manager was running as a internal controller in the `NodeLifecycleManager`, so moving and renaming it as a new top-level controller wouldn't degrade the performance. No bugs are reported in the current implementation.
```

Reply:
That's not necessarily true. Mistakes could have been made when setting the number of workers or something along those lines. But maybe that's too paranoid.

Let's just adjust the wording a little:

```
In the past, the taint-manager was running as an internal controller in the `NodeLifecycleManager`,
so moving and renaming it as a new top-level controller has a low chance of degrading performance.
No bugs are reported in the current implementation.
```

Reply:
updated.